client/blocks/plugins-scheduled-updates/notification-settings.tsx

```diff
@@ -0,0 +1,125 @@
+import { useDispatch, useSelector } from 'calypso/state';
+import { fetchSettings } from 'calypso/state/notification-settings/actions';
+import { getNotificationSettings } from 'calypso/state/notification-settings/selectors';
+import { useSiteSlug } from './hooks/use-site-slug';
+
+import './notification-settings.scss';
+
+type Props = {
+	onNavBack?: () => void;
+};
+
+export const NotificationSettings = ( { onNavBack }: Props ) => {
+	const siteSlug = useSiteSlug();
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+
+	const { data: settings, isFetched } = useScheduledUpdatesNotificationSettingsQuery( siteSlug );
+	const [ formValues, setFormValues ] = useState( {
+		success: false,
+		failure: false,
+	} );
+	useEffect( () => {
+		dispatch( fetchSettings() );
+	}, [ dispatch ] );
+
+	const wpcomNotificationSettings = useSelector( ( state ) => {
+		return getNotificationSettings( state, 'wpcom' );
+	} );
+
+	const { updateNotificationSettings, isPending: isSaving } =
+		useScheduledUpdatesNotificationSettingsMutation( siteSlug );
+
+	useEffect( () => {
+		if ( isFetched && settings ) {
+			setFormValues( {
+				success: settings.success,
+				failure: settings.failure,
+			} );
+		}
+	}, [ isFetched, settings ] );
+
+	const handleCheckboxChange = ( field: keyof typeof formValues ) => ( checked: boolean ) => {
+		setFormValues( ( prevValues ) => ( {
+			...prevValues,
+			[ field ]: checked,
+		} ) );
+	};
+
+	const onSave = useCallback( () => {
+		updateNotificationSettings( formValues );
+	}, [ formValues, updateNotificationSettings ] );
+
+	return (
+		<Card className="plugins-update-manager">
+			<CardHeader size="extraSmall">
+				<div className="ch-placeholder">
+					{ onNavBack && (
+						<Button icon={ arrowLeft } onClick={ onNavBack }>
+							{ translate( 'Back' ) }
+						</Button>
+					) }
+				</div>
+				<Text>{ translate( 'Notification management' ) }</Text>
+				<div className="ch-placeholder"></div>
+			</CardHeader>
+			<CardBody className="notification-settings-form">
+				<label>{ translate( 'Email me' ) }</label>
+				<Text className="info-msg">
+					{ translate(
+						'Receive email notifications to stay informed about the performance of the plugin updates.'
+					) }
+				</Text>
+
+				<div className="form-field">
+					<CheckboxControl
+						label={ translate( 'On successful updates' ) }
+						checked={ formValues.success }
+						onChange={ handleCheckboxChange( 'success' ) }
+						disabled={ ! isFetched || ! wpcomNotificationSettings?.scheduled_updates }
+					/>
+				</div>
+				<div className="form-field">
+					<CheckboxControl
+						label={ translate( 'On failed updates' ) }
+						checked={ formValues.failure }
+						onChange={ handleCheckboxChange( 'failure' ) }
+						disabled={ ! isFetched || ! wpcomNotificationSettings?.scheduled_updates }
+					/>
+				</div>
+
+				{ ! wpcomNotificationSettings?.scheduled_updates && (
```

Comment:
This message shows on loading and disappear later, can we wait until this is fetched before showing the message?